src/Discovery/test/Eureka.Test/EurekaDiscoveryClientTest.cs

```diff
@@ -433,13 +435,13 @@ public async Task GetInstancesByVipAddress_ReturnsExpected()
 
         discoveryClient.Applications = new ApplicationInfoCollection([
             new ApplicationInfo("app1", [
-                new InstanceInfo("id1", "app1", "localhost", "192.168.56.1", new DataCenterInfo(), TimeProvider.System)
+                new InstanceInfo("id11", "app1", "localhost", "192.168.56.1", new DataCenterInfo(), TimeProvider.System)
```

Comment:
Why change the id of these instances? I'm not super concerned about it, just curious

Reply:
Scroll down a bit. The IDs for app2 are named id21/id22, so this change aligns with that pattern.